configuration-service/common/git.go

```diff
@@ -519,32 +525,8 @@ func ServiceExists(project string, stage string, service string, disableUpstream
 
 // GetCredentials returns the git upstream credentials for a given project (stored as a secret), if available
 func GetCredentials(project string) (*common_models.GitCredentials, error) {
```

Comment:
We probably do not have a reason to have this function, as we can use `credentialsReader.GetCredentials(project)`, but as it is configuration-service, there is probably no reason to refactor it and spent time on improving the code